data/en/cfreturn.json

```diff
@@ -11,5 +11,21 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/tag/cfreturn"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/tag/cfreturn"}
 	},
-	"links": []
+	"links": [],
+	"examples": [
+		{
+			"title":"Tag Syntax",
+			"description":"<cfreturn value >",
+			"code":"<cffunction name=\"IncrementByTen\">\r\n<cfargument name=\"num\" required=\"true\">\r\n\r\n<cfset incrementedValue = arguments.num + 10 />\r\n\r\n<cfreturn incrementedValue >\r\n</cffunction>\r\n\r\n<cfoutput>#incrementByTen(30) #</cfoutput>",
+			"result":"40",
+			"runnable":true
+		},
+		{
+			"title":"Script Syntax",
+			"description":"return value;",
```

Comment:
Could you make the description more descriptive here as well?

Reply:
Thanks for pointing it out. I've tried to make titles and descriptions bit more descriptive. I assumed it would be a good idea to keep examples exactly the same switching only the syntax. This means the descriptions are now same in both examples.

Reply:
I guess, that's totally fine.